apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/ExchangeDrawer/index.tsx

```diff
@@ -77,14 +77,15 @@ export default function ExchangeDrawer({ swapTransaction, exchangeRate, onComple
   ) as ExchangeSwap;
 
   const onError = useCallback(
-    (errorResult: { error: Error; swapId?: string }) => {
+    (errorResult: { error: Error | CompleteExchangeError; swapId?: string }) => {
```

Comment:
As CompleteExchangeError extends Error, I don't see the point of this "| CompleteExchangeError"
(I might be wrong, I don't actually know typescript...)

Reply:
No you are right